tools/autotools/tsk_logical_imager.cpp

```diff
@@ -0,0 +1,749 @@
+/*
+ ** tsk_logical_imager
+ ** The Sleuth Kit 
+ **
+ ** Brian Carrier [carrier <at> sleuthkit [dot] org]
+ ** Copyright (c) 2010-2011 Brian Carrier.  All Rights reserved
+ **
+ ** This software is distributed under the Common Public License 1.0
+ **
+ */
+
+#include <iostream>
+#include <string>
+#include <algorithm>
+#include <locale>
+#include <codecvt>
+#include <direct.h>
+#include <winsock2.h>
+#include <locale.h>
+#include <Wbemidl.h>
+#pragma comment(lib, "wbemuuid.lib")
+
+#include <comutil.h>
+
+#include "tsk/tsk_tools_i.h"
+#include "tsk/auto/tsk_case_db.h"
+#include "tsk/img/img_writer.h"
+
+std::wstring GetLastErrorStdStrW();
+std::string GetErrorStdStr(DWORD err);
+std::wstring GetErrorStdStrW(DWORD err);
+
+static TSK_TCHAR *progname;
+
+static std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>> converter;
+
+/**
+* toUpper: convert string to uppercase
+* @param srcStr to convert
+* @return uppercase string
+*/
+string toUpper(const string &srcStr)
+{
+	string outStr(srcStr);
```

Comment:
There's some inconsistent spacing throughout the file. You should change the tabs to spaces.